babynames.py

```diff
@@ -66,15 +87,18 @@ def main(args):
     parser = create_parser()
     # Run the parser to collect command-line arguments into a NAMESPACE called 'ns'
     ns = parser.parse_args(args)
-
     if not ns:
         parser.print_usage()
         sys.exit(1)
 
     file_list = ns.files
-
+    for file_ in file_list:
+        text = extract_names(file_)
+        
```

Comment:
We need to do a conditional logic here... IF there is a flag for --summary_file, then save to the summary file, ELSE print the results.